libs/ebpfnetsh/programs.cpp

```diff
@@ -492,22 +494,22 @@ handle_ebpf_show_programs(
         return status;
     }
 
-    // If the user specified a filename and no level, default to verbose.
-    if (tags[3].bPresent && !tags[2].bPresent) {
+    // If the user specified an ID and no level, default to verbose.
+    if (tags[6].bPresent && !tags[2].bPresent) {
         level = VL_VERBOSE;
     }
 
-    // TODO(#188): We need to implement level, other columns, and implement filtering by attached and pinned.
-
-    std::cout << "\n";
-    std::cout << "    ID            File Name         Section             Name      Mode\n";
-    std::cout << "====== ==================== =============== ================ =========\n";
+    if (level == VL_NORMAL) {
+        std::cout << "\n";
+        std::cout << "    ID  Pins  Links  Mode       Name\n";
```

Comment:
Do we care about localization of these strings?

Reply:
Eventually yes, but it's more than just this one place.   We could file an issue but it's probably low priority.

Reply:
Issue #558